packages/nimble-components/src/patterns/button/tests/doc-strings.ts

```diff
@@ -41,7 +41,7 @@ export const iconDescription = `When including an icon, set \`slot="start"\` on
 
 <details>
     <summary>Icon Usage</summary>
-    Icons should be consistent and only used when necessary.
+    <p>Icons should be consistent and only used when necessary.</p>
```

Comment:
Can you give the same treatment to our remaining usages of summary / detail in other stories. Makes it more likely folks will copy the right pattern in the future.

Reply:
Updated the remaining problematic ones.

Basically it looks like if there's a line of text then `<details>` or `<summary>` as the immediate next line, that triggers the error (as they all get combined into a single `<p>`). If there's a blank line inbetween it becomes a separate paragraph from the `details`/`ul` so it's fine.

I went with the newline approach for text already written as template literal strings since I thought that it's easier to read than the explicit `<p></p>` tags.

Reply:
Can I instead advocate for always having the explicit `<p></p>` tags? It's easier in review to always check the child text is wrapped in a tag then to try and figure out when it is or is not necessary.

> Makes it more likely folks will copy the right pattern in the future.
> > I went with the newline approach for text already written as template literal strings since I thought that it's easier to read than the explicit `<p></p>` tags.

My goal was consistency, I still think the newline approach will be easy for people to copy and paste tweak and mess-up because there are inconsistent patterns used to copy and paste.

Reply:
Unmarking resolve

Reply:
Discussed this some offline. The issue is a bit more complicated than just 'ensure text inside `<details>` is in a `<p>` tag'. The warning can be printed when not using `<details>` (e.g. text then `<ul>`), and plain text in `<details>` is fine (unless again it's followed by something like `<ul>`).

Settled on filing a new issue ( #1393 ) that documents the warnings with some examples + fixes.